content/blog/tikv-3.0ga-release-notes.md

```diff
@@ -0,0 +1,164 @@
+---
+title: TiKV 3.0 GA Release & Benchmarks
+date: 2019-07-08
+---
+
+Today we're proud to announce the general availability of TiKV 3.0! Before release, TiKV 3.0 underwent a rigorous testing regime, including the [official Jepsen test](https://www.pingcap.com/blog/tidb-passes-jepsen-test-for-snapshot-isolation-and-single-key-linearizability/) with TiDB.
```

Comment:
Saying that "before release tikv was tested" as the very first thing I think sends a bad intuition, like "oh, they weren't testing this stuff before". Better to say more simply that we're proud that this time we've finally been honored with a set of jepsen tests and they resulted in huge improvements to the product.

Reply:
Altering it to say we used an *even more* rigorous testing regime to stress we were before.